demo/api/src/importer/data-streams/file-data-stream.ts

```diff
@@ -13,17 +13,25 @@ export interface FileStreamAndSize {
     fileSize?: number;
 }
 
+export type FileStreamAndMetadata = {
+    dataStream: FileStream;
+    metadata: {
+        additionalData: {
```

Comment:
We probably can remove this additional level since metadata already implies that it's additional data.